doc/en-us/dev/js/dev_faq.md

```diff
@@ -18,17 +18,17 @@ PowerNukkitX.listenEvent("cn.nukkit.event.entity.EntityDamageByEntityEvent", Eve
 })
 ```
 
-## 2. Server.getLevel返回null  
+## 2. Server.getLevel returns null
 
-通常是由于以下两个问题造成的：  
-- 在`main`函数开始执行之前调用
-- `plugin.yml`中`load`字段填写了`STARTUP`而非正确的`POSTWORLD`
+This is usually caused by two problems:
+- Calling the `main` function before it starts executing
+- The `load` field in `plugin.yml` is filled with `STARTUP` instead of the correct `POSTWORLD`
 
-## 3. 导入其他JS插件的模块时的一些注意事项
-PNXJS的每个JS插件都是一个独立的`Context`，这意味着，如果你使用`import`导入其他插件的模块时，会完全重新载入一次被导入的插件的模块。  
-例如以下基于[LLSE-Lib](https://github.com/PowerNukkitX/LLSE-Lib)库编写的示例:  
+## 3. Some considerations when importing modules from other JS plugins
+Each JS plugin of PNXJS is a separate `Context`, which means that if you import a module of another plugin using `import`, it will completely reload the module of the plugin being imported once.  
```

Comment:
The Chinese here has context ellipsis, but if they are not supplemented when translated into English, it will lead to incorrect expression of meaning.
The complete sentence goes like this: PNXJS的每个JS插件都是处于一个独立的`Context`之中的
So if we translate like this it may be better: Each PNX-JS plugin runs in a separate `Context`

Reply:
fix in next commit

Reply:
fix in this commit c726061604c73240c6e1dbc554a7eeb6fcfc7182